app/code/Magento/ConfigurableProduct/Controller/Adminhtml/Product/Initialization/Helper/Plugin/CleanConfigurationTmpImages.php

```diff
@@ -0,0 +1,127 @@
+<?php
```

Comment:
Please, add next declaretion and types to methods arguments/returned values where it possible
`declare(strict_types=1);`